src/output.rs

```diff
@@ -188,6 +193,19 @@ impl OutputTypeDetection for TxOut {
         false
     }
 
+    /// Checks if an output pays to a P2A script.
+    ///
+    /// A P2A output pushes the witness version 1 followed by the 2 bytes hex-encoded as 4e73
+    /// `script_pubkey: [ OP_PUSHNUM_1  <4e73> ]`
+    fn is_p2a(&self) -> bool {
+        let script_pubkey_bytes = self.script_pubkey.as_bytes();
+        return script_pubkey_bytes.len() == 4
+            && script_pubkey_bytes[0] == opcodes::OP_PUSHNUM_1.to_u8()
+            && script_pubkey_bytes[1] == opcodes::OP_PUSHBYTES_2.to_u8()
+            && script_pubkey_bytes[2] == 78u8
+            && script_pubkey_bytes[3] == 115u8;
```

Comment:
nit

```suggestion
            && script_pubkey_bytes[2] == 0x4eu8
            && script_pubkey_bytes[3] == 0x73u8;
```

Reply:
Done. If you release I can update `transactionfee-info`.